src/modules/api.js

```diff
@@ -2,14 +2,23 @@ import DisplayCards from "./listCards.js";
 
 export default class Api {
   constructor() {
-    this.artworksList = 'https://api.artic.edu/api/v1/artworks?page=2';
+    this.artworksList = 'https://api.artic.edu/api/v1/artworks?';
```

Comment:
`https://api.artic.edu/api/v1/artworks?page=2` Kindly put the page 2 again so you can see the proper item detail 😅